libraries/consul_installation_binary.rb

```diff
@@ -86,11 +87,14 @@ def consul_program
       end
 
       def self.binary_basename(node, resource)
+        new_filename = node['consul']['enterprise'] ? 'consul-enterprise' : 'consul'
```

Comment:
Is there any reason to use `new_filename` over `filename` ?

Reply:
I can use the shorter nomenclature.

Reply:
It has been updated.

Reply:
Was just trying to understand the importance of `new_`